src/include/btmem.h

```diff
@@ -144,20 +144,25 @@ struct __wt_delta_header {
      */
     uint32_t mem_size; /* 00-03: in-memory size */
 
-    uint8_t version; /* 04: version */
+    union {
+        uint32_t entries; /* 04-07: number of cells on page */
+        uint32_t datalen; /* 04-07: overflow data length */
+    } u;
+
+    uint8_t version; /* 08: version */
 
-    uint8_t type; /* 05: page type */
+    uint8_t type; /* 09: page type */
 
-    uint8_t flags; /* 06: flags */
+    uint8_t flags; /* 10: flags */
 
-    uint8_t distinguished; /* 07: cannot be zero, distinguishes from a full-page image */
+    uint8_t distinguished; /* 11: cannot be zero, distinguishes from a full-page image */
```

Comment:
We can get rid of this field - I think we might redo the block manager header, so we won't have the problem that this would fix.  We can get into details when we meet.

Reply:
ok